ontology/yaml/resources/HVAC/entity_types/CMP.yaml

```diff
@@ -38,3 +38,13 @@ CMP_SS_WDPM:
   - run_mode
   - schedule_run_command
   - control_status
+
+CMP_SS_ZTO:
```

Comment:
shouldnt include an acronym that isnt a defined abstract type. ZTO is undefined.

Reply:
I see. Thanks sir. noted on this and will change it to non-canonical

Reply:
removing this now as we will be using CMP_SS only